### ISSUE TITLE
Question About CAT

### ISSUE THREAD

**Issue by Huai-qIu-tec** [eeyhsong/EEG-Conformer#4]
Hi!
I'm impressed with your visualization method CAT in paper. After reading the paper, I got some questions about CAT and model accuracy.

Q1. In the visualization file CAT.py, the inputs of the file is 'data', is this 'data' from original train data which shape is (288, 1, 22, 1000) after standardization or is it generated in some middle process?
`data = np.load('./grad_cam/train_data.npy') `

Q2. After running the conformer several times, i found the accuracy suprisingly lower than the paper results, so would it possible to share the pre-trained weights of the conformer?

Thank you for sharing your excellent work.
Best Wishes !

**Comment by eeyhsong** [comment]
hello @Huai-qIu-tec,
Thanks for your interest!
- yes, the input shape is still (b, 1, 22, 1000), without other processes.
- We don't use pre-training in this work, just subject-specific training. You may try to check the input data and the output. 
Best. 🤝

**Comment by Huai-qIu-tec** [comment]
Thank you for answering. I'll try the CAT again.
